_pages/how-we-work/how-to-log-in.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Logging in
+tags:
+- ent
+- OAuth
+- Logging
+- login
+- access
+---
+
+This content provides best practices for logging into your various government accounts and apps.  The official GSA Policy can be found [here](https://insite.gsa.gov/portal/content/526805).
+
+## ENT (enterprise) login
+
+This is your account for signing into systems administered by GSA, including your email, calendar, and Google Drive.  Your ENT account name is in the form FirstnameMiddleinitialLastname (Ex: JohnASmith).  For guidance on secure passwords, reference this [page](/password-requirements).
+
+When you login with ENT while not connected to a GSA wireless network, you will be prompted to select "This is a public computer" or "This is a private computer" (see image below).  You should always select "This is a private computer" unless you are logging in on a computer that's not your government computer or a computer that you personally own (Ex: a library computer).  
+
+![addressbar](/images/ENT-login-screen.png)
```

Comment:
this didn't load on the preview, fyi

Reply:
any idea why this isn't loading?  It shows up when I do the preview inside of the github webapp...

Reply:
screenshot? What webapp?

Reply:
i just meant when i do the preview in the github https://github.com/18F/handbook/blob/c4034da3a59659a5ecd27dff5c55c196d835872f/_pages/how-we-work/how-to-log-in.md